datafusion/sqllogictest/test_files/dictionary.slt

```diff
@@ -148,7 +148,7 @@ select count(*) from m1 where tag_id = '1000' and time < '2024-01-03T14:46:35+01
 ----
 10
 
-query RRR
+query RRR rowsort
```

Comment:
rowsort is added because I got
```
700 1000 850
100 600 350
```

Reply:
Yeah --  also added the same fix in https://github.com/apache/arrow-datafusion/pull/8769 and I agree this looks good